test_scripts/Polices/Related_HMI_API/ATF_OnAllowSDLFunctionality_allowed_false_without_device.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     1. Preconditions: App is registered
+--     2. Steps: Activate App, send SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' to HMI
+-- Requirement summary: 
+--     [Policy] "EXTERNAL_PROPRIETARY" flow: Related HMI API
```

Comment:
@vvvakulenko The test is checking flow: EXTERNAL_PROPRIETARY.
To be sure that it is correct flow, you should have:

testCasesForBuildingSDLPolicyFlag:Update_PolicyFlag("EXTENDED_POLICY", "EXTERNAL_PROPRIETARY")
testCasesForBuildingSDLPolicyFlag:CheckPolicyFlagAfterBuild("EXTENDED_POLICY","EXTERNAL_PROPRIETARY")

You need to have file: testCasesForBuildingSDLPolicyFlag.lua from https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/32
Please note that this functions are reviewed locally because should be sure that SDL is build and after that working normally.